components/script/dom/globalscope.rs

```diff
@@ -203,7 +203,7 @@ pub(crate) struct GlobalScope {
     broadcast_channel_state: DomRefCell<BroadcastChannelState>,
 
     /// The blobs managed by this global, if any.
-    blob_state: DomRefCell<BlobState>,
+    blob_state: DomRefCell<HashMapTracedValues<BlobId, BlobInfo>>,
```

Comment:
If you're going to remove the `BlobState`, the meaning an empty hashmap needs to be documented.